extensions-okd-c9s.yaml

```diff
@@ -3,7 +3,6 @@
 # and https://github.com/coreos/fedora-coreos-tracker/issues/401
 
 repos:
-  - c9s-sig-virtualization
```

Comment:
```
error: Packages not found: openvswitch3.5-ipsec
```
Looks like we need to add `c9s-sig-nfv` here

Reply:
Should be fixed now.